doc/languages.rst

```diff
@@ -389,6 +390,11 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check syntax and types of Go tests with the `Go compiler`_.
 
+      .. option:: flycheck-go-build-install-deps
+
+         Whether to install dependencies while checking. **Note** this option
+         also affects `go-build`
```

Comment:
Don't repeat the documentation here.  Instead, with `:noindex:` as in the comment above, just say ``See `flycheck-go-build-install-deps`.`` and it'll cross-reference the real docs.